bonita-connector-rest-def/src/main/resources/RESTConnector-3.0.def

```diff
@@ -42,7 +42,11 @@
   <input name="proxy_port" type="java.lang.String"/>
   <input name="proxy_username" type="java.lang.String"/>
   <input name="proxy_password" type="java.lang.String"/>
-  <output name="result" type="org.bonitasoft.connectors.rest.RESTResult"/>
+  <output name="body" type="java.lang.String"/>
+  <output name="headers" type="java.util.List"/>
+  <output name="execution_time" type="java.lang.Double"/>

```

Comment:
I thought it could be useful to provide this information, it can be removed if not.